docs/release_notes/2024/august2024.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+id: August_2024
+description: Release notes for August
+title: August 2024
+tags:
+  - release notes
+  - changelog
+  - august
+---
+
+## 3.3.9.\* (August 2, 2024)
+
+- Prophecy Python libs version: 1.9.9
+- Prophecy Scala libs version: 8.0.29
+
+### Features {#Features339}
+
+#### Spark Copilot Enhancements
+
+- **Snowflake Partner Network Integration**: We've added a Connection String Identifier for Snowflake, following the Snowflake Native Connector Best Practices Guide. This was a requirement to advance in the Snowflake Partner Network.
+
+- **Support custom artifactory resolve plibs version**: We now support custom artifactory resolution mode on EMR Fabrics such as Livy.
```

Comment:
This is not correct. 

Support Custom dependencies for Livy: We now support adding custom Pypi or maven dependencies for pipelines running on all spark Fabrics including EMR, Dataproc, etc

Reply:
Done.